src/App.vue

```diff
@@ -389,6 +393,19 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
     return crumbs
   }
 
+  /** The page header (title). */
+  get header (): string {
+    if (this.isAmalgamationFilingRegular) {
+      return `${this.getFilingName} (Regular)`
+    } else if (this.isAmalgamationFilingHorizontal) {
+      return `${this.getFilingName} (Horizontal Short-form)`
+    } else if (this.isAmalgamationFilingVertical) {
+      return `${this.getFilingName} (Vertical Short-form)`
+    } else {
+      return this.getFilingName
+    }
+  }
```

Comment:
This text is specific to the app title so I didn't move it into the store.

Reply:
Wouldn't it be a good idea to those names to the FilingNames shared enum Sev?

Reply:
This exact text is only used here, so I don't think there's a good reason to move it there.